keps/sig-scheduling/3022-min-domains-in-pod-topology-spread/README.md

```diff
@@ -307,12 +316,16 @@ Longer term, we may want to require automated upgrade/rollback tests, but we
 are missing a bunch of machinery and tooling and can't do that now.
 -->
 
+Yes. The behavior is changed as expected.
```

Comment:
Describe the manual testing that was done :)

Reply:
I'm ok with this manual testing. Although, a test that most closely follows the question is when you actually do an upgrade.

So you start with an apiserver in version 1.24 (with feature disabled) and then upgrade to 1.25 with feature enabled and back.

Reply:
Okay, I'll do another manual test.